CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Version 1.11.0 (Under active development)
 
+### AppCenter
+
+* **[Fix]** Fix minimum storage size verification to match actual value.
```

Comment:
actual value => minimum possible value?

Reply:
Anything you change here in CHANGELOG.md, please update the other one in Apple SDK repo.
The PR was already merged so you probably need to submit a new PR only for that change.
https://github.com/Microsoft/AppCenter-SDK-Apple/pull/1220